package.json

```diff
@@ -21,6 +21,7 @@
     "babel-core": "6.25.0",
     "babel-eslint": "7.2.3",
     "babel-generator": "^6.26.0",
+    "babel-macros": "1.2.0",
```

Comment:
[`babel-macros` is now `babel-plugin-macros`](https://github.com/kentcdodds/babel-plugin-macros/pull/42).